src/core/varlink.c

```diff
@@ -424,7 +425,13 @@ static int manager_varlink_init_system(Manager *m) {
                         if (!fresh && varlink_server_contains_socket(m->varlink_server, address))
                                 continue;
 
-                        r = sd_varlink_server_listen_address(m->varlink_server, address, 0666 | SD_VARLINK_SERVER_MODE_MKDIR_0755);
+                        if (path_is_absolute(address)) {
+                                r = mkdir_parents_label(address, 0755);
+                                if (r < 0)
+                                        log_error_errno(r, "Failed to create directory '%s': %m", address);
```

Comment:
Please downgrade the log level to warning and mention that the failure is ignored.
Also clarify we failed to create parent directory.
```c
log_warning_errno(r, "Failed to create parent directory of '%s', ignoring: %m", address);
```